modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -14,11 +14,26 @@
 namespace cv {
 namespace gapi {
 
+/**
+ * The enum specified format of result that you get from @ref cv::gapi::stereo.
+ */
 enum class StereoOutputFormat {
-    DEPTH_FLOAT16,
-    DEPTH_FLOAT32,
-    DISPARITY_FIXED16_11_5,
-    DISPARITY_FIXED16_12_4
+    DEPTH_FLOAT16, ///< Floating point 16 bit value, CV_16FC1
+    DEPTH_FLOAT32, ///< Floating point 32 bit value, CV_32FC1
+    DISPARITY_FIXED16_11_5, ///< 16 bit signed: first bit for sign,
+                            ///< 10 bits for integer part,
+                            ///< 5 bits for fractional part.
+    DISPARITY_FIXED16_12_4, ///< 16 bit signed: first bit for sign,
+                            ///< 11 bits for integer part,
+                            ///< 4 bits for fractional part.
+    DEPTH_F16, ///< Floating point 16 bit value, CV_16FC1
+    DEPTH_F32, ///< Floating point 32 bit value, CV_32FC1
+    DISPARITY_Q16_10_5, ///< 16 bit signed: first bit for sign,
+                        ///< 10 bits for integer part,
+                        ///< 5 bits for fractional part.
+    DISPARITY_Q16_11_4  ///< 16 bit signed: first bit for sign,
+                        ///< 11 bits for integer part,
+                        ///< 4 bits for fractional part.
```

Comment:
No need to comment it separately, just put `///< Same as TYPE_NAME`.

Reply:
Done